packages/components/skeleton/src/skeleton.scss

```diff
@@ -0,0 +1,70 @@
+:host {
+  --background: rgb(var(--sl-color-palette-neutral-150));
+  --border-radius: 0.25rem;
+  --min-height: 0.5rem;
+  --shine-color: rgb(var(--sl-color-palette-neutral-200));
+  --animation-duration: 3000ms;
+}
+
+:host {
+  background: var(--background);
+  border-radius: var(--border-radius);
+  display: block;
+  min-height: var(--min-height);
+}
+
+:host([effect='shimmer']) {
+  animation: shimmer-effect var(--animation-duration) linear infinite;
+  background: linear-gradient(to right, var(--background) 4%, var(--shine-color) 25%, var(--background) 36%);
+  background-size: 2000px 100%;
+}
+
+:host([effect='sheen']) {
+  animation: sheen-effect var(--animation-duration) linear infinite;
+  background-image: linear-gradient(
+    120deg,
+    var(--shine-color),
+    var(--background),
+    var(--background),
+    var(--shine-color)
+  );
+  background-size: 400% 100%;
+}
+
+:host([effect='pulse']) {
+  animation: pulse-effect var(--animation-duration) ease-in-out 0.5s infinite;
+}
+
+@keyframes shimmer-effect {
+  0% {
+    background-position: -2000px 0;
+  }
+
+  100% {
+    background-position: 2000px 0;
+  }
+}
+
+@keyframes sheen-effect {
```

Comment:
this animation makes me a bit nervous :D it's so fast compared to the other ones, even though they share the "animation-duration".